### PR TITLE
TVR-16999 Request API v4 document update

### PR DESCRIPTION
**Pull request by quancheng-zhao**
Update RaaS service docs on the latest changes.

### CODE REVIEW

File: src/api-reference/request/v4.endpoints.request-resources.markdown [hunk 1]
```diff
@@ -571,7 +571,7 @@ PUT {datacenter}/travelrequest/v4/requests/{requestUuid}
 Name|Type|Format|Description
 ---|---|---|---
 `requestUuid`|`string`|-|**Required** The unique identifier of the Request.
-`userId`|`string`|-|The unique identifier of the user performing the update. **Required** when connecting with a Company token. If empty a 400, `missingRequiredParam` error code will be displayed.
+`userId`|`string`|-|The unique identifier of the user performing the update. Optionnal when connecting with a Company token.
```
Comment (laurentlhermine) [inline src/api-reference/request/v4.endpoints.request-resources.markdown:574]: Optional ?

Comment (laurentlhermine) [reply src/api-reference/request/v4.endpoints.request-resources.markdown:574]: We could add that it is automatically provided and will be ignored with a user token.

Comment (cbobrie) [reply src/api-reference/request/v4.endpoints.request-resources.markdown:574]: add the line "Only used when connecting with a Company token, optional." - @laurentlhermine to confirm phrasing

Comment (laurentlhermine) [reply src/api-reference/request/v4.endpoints.request-resources.markdown:574]: Suggesting the following :

`The unique identifier of the user performing the update. Optional. Will be taken into account only if calling with a Company token. If not provided the update will be performed as "Concur System".`

**Comment by quancheng-zhao** [comment]
Hello @jonikamelcher , could you merge this pull request, please? Thank you

**Review by RoAnnC1 — APPROVED** [review]
:shipit:

**Pull request merged**